backend/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import express, { Application } from "express";
+import dotenv from "dotenv";
+import { WordsController } from "./controllers/words/words.controller";
+import cors from "cors";
+import router from "./routes/words/words.route";
+import { SetsController } from "./controllers/sets/sets.controller";
```

Comment:
it might be a good idea to keep imports organised, you can install `eslint` or `prettier` that will do the job automatically for you. common practice is to keep external libs at the top, then line of empty space, and imports from the project so the reader's eye catches quicker what is important to them

Reply:
resolved